.gitignore

```diff
@@ -1,4 +1,5 @@
 HELP.md
+apllication.properties
```

Comment:
we don't want to ignore apllication.properties

Reply:
I have added application.properties

Reply:
Please, remove it from ```.gitignore``` file